packages/plugin-core/package.json

```diff
@@ -1429,6 +1429,17 @@
     ]
   },
   "icon": "media/logo.png",
+  "binary": {
+    "module_name": "node_sqlite3",
+    "module_path": "./lib/binding/napi-v{napi_build_version}-{platform}-{libc}-{arch}",
+    "host": "https://github.com/TryGhost/node-sqlite3/releases/download/",
+    "remote_path": "v5.1.2",
+    "package_name": "napi-v{napi_build_version}-{platform}-{libc}-{arch}.tar.gz",
+    "napi_versions": [
+      3,
```

Comment:
nitpick but do you know if we need to support node api verison 3 if our min node version is set to 16?

Reply:
removed